engine/src/main/java/io/zeebe/engine/processing/streamprocessor/ReProcessingStateMachine.java

```diff
@@ -211,11 +220,11 @@ private void readNextEvent() {
     }
 
     currentEvent = logStreamReader.next();
-    if (currentEvent.getPosition() > lastSourceEventPosition) {
+    if (currentEvent.getPosition() > lastFollowUpEventPosition) {
       throw new IllegalStateException(
           String.format(
               ERROR_MESSAGE_REPROCESSING_NO_SOURCE_EVENT,
```

Comment:
Here the error message still says  "source event".

Reply:
True. I can change it to `"Expected to find last follow-up event position '%d', but last position was '%d'. Failed to reprocess on processor"`

Reply:
:+1: